src/main/include/subsystems/ColorSensorSubsystem.h

```diff
@@ -6,28 +6,21 @@
 
```

Comment:
This has been irrelevant for a while and should be removed

Reply:
this has been removed